lasagne/tests/layers/test_normalization.py

```diff
@@ -101,8 +101,8 @@ def input_layer(self, input_data):
     @pytest.fixture
     def layer(self, input_layer):
 
-        from lasagne.layers.normalization import\
-                LocalResponseNormalization2DLayer
+        from lasagne.layers.normalization import (
+            LocalResponseNormalization2DLayer,)
```

Comment:
Same issue here 
/home/joncrall/code/Lasagne/lasagne/tests/layers/test_normalization.py:105:17: E126 continuation line over-indented for hanging indent
                LocalResponseNormalization2DLayer
                ^
/home/joncrall/code/Lasagne/lasagne/tests/layers/test_normalization.py:122:17: E126 continuation line over-indented for hanging indent
                LocalResponseNormalization2DLayer

I just removed the \ while I was working on that pep8 error.